### ISSUE TITLE
Bounding Box extraction during training 

### ISSUE THREAD

**Issue by gorjul** [ultralytics/yolov5#3749]
## ❔Question
Is it possible to extract the bounding boxes during training inside the ComputeLoss class?

```
pxy = ps[:, :2].sigmoid() * 2. - 0.5
pwh = (ps[:, 2:4].sigmoid() * 2) ** 2 * anchors[i]
pbox = torch.cat((pxy, pwh), 1)  # predicted box
```

This creates an anchor box if I am not mistaken? The thing I struggle with is to get bounding box coordinates of these anchor boxes then.

**Comment by glenn-jocher** [comment]
@gorjul these coordinates are expressed in grid units (i.e. 0-19), not in pixel units, i.e. 0-639.

If you want box crops you can obtain them directly from a trained model:
```
python detect.py --weights path/to/model.pt --source path/to/images --save-crop
```

**Comment by jaideep11061982** [comment]
what is format in which bbx are saved  xywh or xcycwh  @glenn-jocher

**Comment by glenn-jocher** [comment]
@jaideep11061982 label format is discussed in Train Custom Data Tutorial, start there:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
